contracts/Vault.vy

```diff
@@ -1,4 +1,4 @@
-# @version 0.2.16
+# @version 0.3.0
```

Comment:
v0.3.1 should be released today, should probably try that out

Reply:
IncompatibleVyperVersion: No installable vyper version matching '0.3.1' in 'contracts/Vault.vy'